### ISSUE TITLE
Installer job fails with some authentication error

### ISSUE THREAD

**Issue by tkolhar** [SatelliteQE/automation-tools#316]
Hi,

 The installer job get stuck at below error:
[abc.redhat.com] run: ssh libvirt1.redhat.com date
[abc.redhat.com] out: The authenticity of host 'libvirt1.redhat.com (x.x.x.x)' can't be established.

Parameters passed
SERVER_HOSTNAME hostname
DISTRIBUTION Downstream
SATELLITE_VERSION 6.2
SELINUX_MODE enforcing
PROXY_MODE authenticated
FIX_HOSTNAME Yes
PARTITION_DISK Yes

**Comment by kbidarkar** [comment]
I am not 100% sure, but this could also be a DNS issue and needs to be investigated.
For now one can just stop this ongoing process and continue with the box. This task is for configuring External libvirt boxes.

**Comment by lpramuk** [comment]
On that box there were mismatch between public key a our deployed private key.
A confession has been made that this conflicting public key was created during manual play with ssh-keygen :-)

We can make setup_libvirt_key task more robust. 
I volunteer :-)

**Comment by elyezer** [comment]
To me this issue is not valid, that was an random error. In the previous runs everything went fine and the `setup_libvirt_key` completed successfully.

In the only job I saw the issue on the history I found out that for some reason the password was being asked:

```
[machine] out: root@libvirt1.satellite.EDITED.redhat.com's password: ...
```

But it has been ~29 builds (most of them green) after it and it didn't happened anymore.
